openprescribing/matrixstore/README.md

```diff
@@ -115,17 +115,91 @@ matrix = deserialize(value)
 print(matrix)
 ```
 
+### The MatrixStore class
+
+The [matrixstore.open](./open.py) module contains a `MatrixStore` class
+which, among other things, automatically handles deserialzation for you.
+
+Initialise it like this:
+```python
+from matrixstore.open import MatrixStore
+matrixstore = MatrixStore.from_file('/path/to/file.sqlite')
+
+# Or with an existing sqlite connection like this:
+# matrixstore = MatrixStore(sqlite_connection)
+```
+
+And query it like this:
+```python
+results = matrixstore.query(
+    'SELECT bnf_code, items FROM presentation WHERE bnf_code LIKE ?',
+    ['10%']
+)
+for items, quantity in results:
+    print(bnf_code, items)
+```
```

Comment:
When we have code samples in the README, could we include the output too?

Reply:
This is a good suggestion, but I can't really face doing this right now.